ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -78,6 +78,61 @@ objects:
 
     containers:
 
+    - name: lease
+      image: ci/boskoscli:latest
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          cpu: 10m
+          memory: 10Mi
+        limits:
+          memory: 200Mi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: CLUSTER_TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'kill $(jobs -p); exit 0' TERM
```

Comment:
I don't think you want to hard-code `exit 0` here.  Can you follow [this example][1] and use:

```sh
trap 'rc=$?; if test "${rc}" -ne 0; then touch /tmp/exit; fi; exit "${rc}"' EXIT
trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
```

[1]: https://github.com/openshift/release/blob/44dd3116723fe2b4ec7ea4a8317aa48a418fd7b6/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L660-L661

Reply:
I didn't know if we wanted this container to be best-effort or not, since `setup` was?

Reply:
(updated to follow the other approach)